[2023-01-30T15:49:39.717+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-01-30T15:49:38.448413+00:00 [queued]>
[2023-01-30T15:49:39.729+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-01-30T15:49:38.448413+00:00 [queued]>
[2023-01-30T15:49:39.730+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-01-30T15:49:39.731+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-01-30T15:49:39.732+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-01-30T15:49:39.788+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-01-30 15:49:38.448413+00:00
[2023-01-30T15:49:39.812+0000] {standard_task_runner.py:55} INFO - Started process 2765 to run task
[2023-01-30T15:49:39.865+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'manual__2023-01-30T15:49:38.448413+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpgzu2fhxe']
[2023-01-30T15:49:39.882+0000] {standard_task_runner.py:83} INFO - Job 18: Subtask create_table
[2023-01-30T15:49:40.148+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table manual__2023-01-30T15:49:38.448413+00:00 [running]> on host 3c28126f9714
[2023-01-30T15:49:40.370+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-01-30T15:49:38.448413+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-30T15:49:38.448413+00:00
[2023-01-30T15:49:40.390+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-01-30T15:49:40.396+0000] {sql.py:315} INFO - Running statement: 
        create table if not exists users(
            firstname text not null, 
            lastname text not null, 
            country text not null, 
            username text not null, 
            password text not null,
            email text not null);
            , parameters: None
[2023-01-30T15:49:40.401+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-01-30T15:49:40.424+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230130T154938, start_date=20230130T154939, end_date=20230130T154940
[2023-01-30T15:49:40.490+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-01-30T15:49:40.519+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
